save_data.rb

```diff
@@ -0,0 +1,44 @@
+require 'json'
+
+module DataSaver
+  def get_data(file)
+    path_file = "./store_data/#{file}.json"
+    File.new(path_file, 'w+') unless File.exist?(path_file)
+    File.write(path_file, '[]') if File.empty?(path_file)
+    file_contents = File.read(path_file)
+    JSON.parse(file_contents)
+  end
```

Comment:
- [ ] Data is saved to JSON before the app exits. In this activity we want a user data to be saved only when the app exits. Kindly ensure to improve upon this aspect.

Reply:
Hi, @rloterh!!!

Thanks for the thorough review provided. We started working on this but found out that what we did was inside the requirements. Unless we didn't understand well. It was requested from us to save data when exiting the app. And load it when starting the app
![save_data](https://user-images.githubusercontent.com/29674658/213157519-ad4823bc-a057-4634-b19d-fa895bed4af7.png)

Please help us understand this more if we are mistaking. Thanks

Reply:
Hello @Samuellekpor 
This simply means that:
Data should be saved to JSON only when option 7 (Exit app) is selected!

Cheers! 😃